content/manager/bootstrapping.md

```diff
@@ -165,6 +165,97 @@ Services:
 {{< /gsHighlight >}}
 
 
+# Deploying a Manager Image
+
+{{% gsNote title="Prerequisites" %}}
+ * A manager image built using the [Image Builder](https://github.com/cloudify-cosmo/cloudify-image-bakery/tree/master/cloudify_marketplace)
+ * For the command-line install, the [`cfy` command]({{< relref "/cli/overview" >}})
+ * Account credentials for the platform you are deploying on
+{{% /gsNote %}}
+
+
+ 1. Create an instance of the [image builder](https://github.com/cloudify-cosmo/cloudify-image-bakery/tree/master/cloudify_marketplace) image
+ 2. Make a note of the IP/hostname
+
+    ```bash
+    $ CLOUDIFY_HOST={your-manager-ip}
+    ```
+
+
+## Command Line
+
+ 1. Switch to the new manager:
+
+    ```bash
+    $ export CLOUDIFY_USERNAME=cloudify
+    $ export CLOUDIFY_PASSWORD=cloudify
+    $ export CLOUDIFY_SSL_TRUST_ALL=true
+    $ cfy use --port 443 --management-ip ${CLOUDIFY_HOST}
+    ```
+
+    {{% gsNote title="TLS" %}}
+    If you have disabled security then use port 80 instead (in this case the `export` lines above are not required): `$ cfy use --port 80 --management-ip ${CLOUDIFY_HOST}`.
+
+    {{% /gsNote %}}
+
+ 2. Create an input file for the `CloudifySettings` blueprint
```

Comment:
I'd rather we provided the webui method first.. and then write (at the end) that if you want to use the CLI instead.. here's what you do. The default should be the webui